.github/workflows/lock.yml

```diff
@@ -0,0 +1,21 @@
+name: 'Lock Threads'
+
+on:
+  schedule:
+    - cron: '50 1 * * *'
```

Comment:
Not related to the PR, but I would like to know how to avoid GitHub token rate limit throttling since we have the same cron expression in all repositories, and every night I receive a lot of emails when this workflow fails.

There is an open issue already, but it doesn't seem to be moving anywhere - https://github.com/dessant/lock-threads/issues/35